packages/create-vite/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@vitejs/create-app",
+  "name": "create-vite",
   "version": "2.4.5",
```

Comment:
Do we need to reset the version?

What effects will this have on the changelogs?

Reply:
I don't think we need to reset the version, as it is just a rename.

Looks like the changelog will look something like
```
## 2.4.6 (2021-07-08)

## [2.4.5](https://github.com/vitejs/vite/compare/create-app@2.4.4...create-app@2.4.5) (2021-07-05)

...
```
For the first release. And from then on, it should properly give us back the link in the titles. The above changeset was generated by a release dry run.
So we should be fine here, and we can PR some notes in 2.4.6 (or 2.5?) about the rename